foundation_cms/templates/patterns/pages/nothing_personal/article_page.html

```diff
@@ -17,7 +17,7 @@
                     </div>
                     <div class="cell small-12 medium-3 medium-offset-2">
                         <div class="article-meta-published">
-                            <span class="published-label">Published</span>
+                            <span class="published-label">{% trans "Published" %}</span>
                             <p class="published-date">{{ page.first_published_at|date:"F j, Y" }}</p>
```

Comment:
We need to switch to a predefined format as only those are automatically localized. Here is the full ocumentation: https://docs.djangoproject.com/en/5.2/ref/settings/#date-format
```suggestion
                            <p class="published-date">{{ page.first_published_at|date:"DATE_FORMAT" }}</p>
```